hw/ip/adc_ctrl/doc/programmers_guide.md

```diff
@@ -45,13 +45,14 @@ The debug accessory voltage range of interest is shown in the diagram below:
 The ADC can be used to detect debug cable connection / disconnection in the non-overlapping regions.
 As an example use case of the two channel filters they can be used for detection of a USB-C debug accessory.
 The ADC must meet some minimum specifications:
-* Full scale range is 0.0V to 2.2V
-* If the signal is below 0.0V the ADC value will be zero.
-* If the signal is above 2.2V the ADC value will be maximum (i.e. same as 2.2V)
-* Absolute maximum error +/- 15 mV in the 0.25 - 0.45 V range
-* Absolute maximum error +/- 30 mV in the rest of the 0.0 - 2.2 V range
+* Full scale range must be 0.0V to 2.2V (note: the ADC in Earlgrey has a full range of 0.0V - 2.3V)
+* If the signal is below 0.0V the ADC value must be zero.
+* If the signal is above 2.2V the ADC value must be maximum (i.e. same as 2.2V)
+* Absolute maximum error +/- 15 mV must be in the 0.25 - 0.45 V range
```

Comment:
Nit: I think this would be better with the must be earlier?

```suggestion
* Absolute maximum error must be +/- 15 mV in the 0.25 - 0.45 V range
```

Reply:
ah yes - what I wrote did not make any sense :D